src/index.js

```diff
@@ -1,14 +1,43 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const tasksSection = document.querySelector('.task-section');
+
+const tasks = [
+  {
+    description: 'Clean Room',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Go Shopping',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Walk Dog',
+    completed: false,
+    index: 3,
+  },
+];
+
+const itterate = () => {
+  tasks.forEach((task) => {
+    const newTask = document.createElement('div');
+    newTask.classList.add('task-row');
+    const rmvIcon = document.createElement('i');
+    rmvIcon.classList.add('fa-regular');
+    rmvIcon.classList.add('fa-square-full');
+
+    const taskText = document.createElement('p');
+    taskText.innerHTML = `${task.description}`;
+    const editIcon = document.createElement('i');
+    editIcon.classList.add('fa-regular');
+    editIcon.classList.add('fa-pen-to-square');
+    newTask.append(rmvIcon, taskText, editIcon);
+    tasksSection.appendChild(newTask);
+  });
+};
```

Comment:
## [OPTIONAL]

- [ ] As you should follow the template design, Kindly make sure the todo texts are aligned at the left and not at the center, just next to the checkboxes.

![image](https://user-images.githubusercontent.com/65068771/165193089-6f38dbd7-7850-4cf9-8b19-1c84ee7e7c61.png)

Reply:
Thanks for leaving your feedback, the issue was addressed and corrected.